### ISSUE TITLE
Add update command

### ISSUE THREAD

**Issue by phyalexander** [phyalexander/Saturn#7]
This command should check for updates here on github and if there is update download it and install.

This means it should parse the tags and compare it with the version of project already installed.

Also the version of the project must be corrected (i mean the message, returned by 'version' argument of Saturn Tool')

**Comment by phyalexander** [comment]
Maybe this could help:
https://index.scala-lang.org/com-lihaoyi/requests-scala

So that 

```scala
val r = requests.get("https://github.com/phyalexander/Saturn/releases/latest")
val currentVersion = _
if (r.url.split("/").last == currentVersion) {
    println(s"Your version $currentVersion is up to date")
} else {
    // run curl, process installation script
}
```
**The installation script must check if Saturn is in path !!!!**

**Comment by phyalexander** [comment]
Implemented at brach 'release-0.2.0'
